src/rhsm/config.py

```diff
@@ -198,6 +201,15 @@ def set(self, section, name, value):
                 self.add_section(section)
             super(RhsmConfigParser, self).set(section, name, value)
 
+    def is_invalid_default_log_level(self, value):
+        if value not in ['CRITICAL', 'ERROR', 'WARNING', 'INFO', 'DEBUG', 'NOSET']:
+            print("Invalid Log Level: {lvl}, setting to INFO for this run.".format(lvl=value), file=sys.stderr)
+            print(
+                "Please use:  subscription-manager config --logging.default_log_level=<Log Level> to set the default_log_level to a valid value.",
```

Comment:
User usually does not know what valid values are, but it would be easy to print these valid values.

Reply:
added valid values to the message